### ISSUE TITLE
Please add feature/module - LazyLibrarian

### ISSUE THREAD

**Issue by thejrose1984** [Hellowlol/HTPC-Manager#99]
Would like to see LazyLibrarian added as a module to HTPC-Manager

**Comment by thejrose1984** [comment]
Any update/progress?

**Comment by Hellowlol** [comment]
Nope, i dont use LazyLibrarian. Problem with adding stuff is that you have to maintain it as well.. I will accept a pr :)

If anybody is interested in making this module:
Use the hp module as a base for you pr. It will save you alot of work as lazy librarian based on hp.

**Comment by thejrose1984** [comment]
Huh?

**Comment by Hellowlol** [comment]
Lol wtf. Sorry. I have edited my response.
